test/parseComment.js

```diff
@@ -343,6 +343,73 @@ describe('parseComment', function () {
     });
   });
 
+  it('Handle multiple inline type tags in multiline description', function () {
```

Comment:
I added a test for a multiline description (which actually parses to a string without line brakes) and multiple inline type tags.